scripts/publish.sh

```diff
@@ -270,7 +270,7 @@ publish_to_npm () {
         read -p "OTP token? " OTP
     done
 
-    npm publish --tag "${TAG:-latest}" --otp "$OTP"
```

Comment:
This is the only thing that I didn't test it manually. Do you see a better way to do it @nvie ?

Reply:
I've never seen this syntax, but it may work. For the decoders build script, I'm just `cd`ing into that dist directory, and then using this very same `npm publish` command, where it will pick up the local `package.json` file to publish from. Can add!

Reply:
Added in 43180fcc0cefccb564701f23f4e18185aaec979a!